sql/base/auth_database.sql

```diff
@@ -583,6 +583,7 @@ CREATE TABLE `realmlist` (
   `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
   `name` varchar(32) NOT NULL DEFAULT '',
   `address` varchar(255) NOT NULL DEFAULT '127.0.0.1',
+  `address6` varchar(255) NOT NULL DEFAULT '',
```

Comment:
Presumably you want it to bind to both IPv4 and IPv6 interfaces. So best to have the ability to set both addresses.